lib/plugin.js

```diff
@@ -3,6 +3,9 @@ import Axios from 'axios'
 
 // Axios.prototype cannot be modified
 const axiosExtra = {
+  setBaseURL (baseURL, browserBaseURL = null) {
+    this.defaults.baseURL = process.browser && browserBaseURL ? browserBaseURL : baseURL
```

Comment:
I think it would be more readable:

(process.browser && browserBaseURL) ?

Reply:
ok